skycoin/skywire
# Integrate skywire-peering-daemon to mainnet

Fixes #	

 Changes:	
-	Added method to startup skywire-peering-daemon as an external process, and receive packets 
       from the daemon over named pipes
-    Added option to visor config to disable skywire-peering-daemon by default
-   Updated skywire-mainet's `make build` command to build skywire-peering-daemon and copy its 
     binary to ./skywire-mainnet

How to test this PR:
-    Ensure a skywire-services integration env is set 
-    Clone skywire-peering-daemon to a  directory adjacent to skywire-mainnet & skywire-services.
-    Add option  {"auto_start_spd": true} to `./skywire- 
       services/integration/generic/{nodeC.json/nodeA.json}`  to enable the daemon
-    
       # run skywire-services
       $  make integration-build 
       $  make integration-run-generic
